Dockerfile

```diff
@@ -4,7 +4,8 @@ WORKDIR /code
 ENV PATH /opt/conda/bin:${PATH}
 ENV LANG C.UTF-8
 ENV SHELL /bin/bash
-RUN /bin/bash -c "install_packages wget bzip2 ca-certificates git && \
+RUN apt-get update && \
```

Comment:
You probably going to get a remark from codefactor advising against apt-get update otherwise if this works I am okay with it.

Reply:
It didn't seem to trigger! I suppose we will see, haha. It's actually okay if we remove this. I saw that tests passed so I didn't notice issue.